wpimath/src/main/native/cpp/trajectory/Trajectory.cpp

```diff
@@ -124,6 +124,17 @@ Trajectory Trajectory::RelativeTo(const Pose2d& pose) {
   return Trajectory(newStates);
 }
 
+Trajectory Trajectory::operator+(const Trajectory& other) const {
+  auto states = m_states;
+  auto otherStates = other.States();
+  for (auto& otherState : otherStates) {
+    otherState.t += m_totalTime;
+  }
+
+  states.insert(states.end(), otherStates.begin() + 1, otherStates.end());
```

Comment:
Are the final t of the previous trajectory and initial t of the next trajectory guaranteed to be the same?